src/components/VImageDetails/VRelatedImages.vue

```diff
@@ -21,10 +19,10 @@
 import { ref } from '@nuxtjs/composition-api'
 import useRelated from '~/composables/use-related'
 import { IMAGE } from '~/constants/media'
-import ImageGrid from '~/components/ImageGrid/ImageGrid'
+import ImageGrid from '~/components/ImageGrid/ImageGrid.vue'
```

Comment:
Should we switch this to use the new ImageGrid from the search results page?

Reply:
It should definitely change once we have the new grid with masonry layout, but I don't see a new `VImageGrid` component yet. Do you mean the `VImageCell`? It's only for the square layout of all search results I believe.

Reply:
Oops, for some reason I thought the new masonry layout was already merged, but I was thinking of the square layout 🤦‍♀️ 

Sorry for the distraction!